src/components/BackButton/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { useHistory } from 'react-router-dom';
+
+import backIcon from 'images/back-button-white.svg';
+
+const BackButton = () => {
```

Comment:
Making `BackButton` a component as there are two places currently using the same button. One in `ForgotPassword` and the other one in `SurveyDetail/detail.js`.